packages/mobile/src/qrcode/QRGen/QRGen-matrix.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { genMatrix } from 'src/qrcode/QRGen'
```

Comment:
No need for a new file here, merge it with the QRGen.test.tsx file and put it under a separate `describe` block.
Also, if a file doesn't contain jsx code, the filename should be just .ts